x/gov/keeper/querier_test.go

```diff
@@ -237,11 +205,6 @@ func TestQueries(t *testing.T) {
 	require.Equal(t, deposit2, deposits[0])
 	require.Equal(t, deposit4, deposits[1])
 
-	deposit = getQueriedDeposit(t, ctx, keeper.cdc, querier, proposal2.ProposalID, TestAddrs[0])
-	require.Equal(t, deposit2, deposits[0])
-	deposit = getQueriedDeposit(t, ctx, keeper.cdc, querier, proposal2.ProposalID, TestAddrs[1])
-	require.Equal(t, deposit4, deposits[1])
```

Comment:
I don't understand why this was removed (probably should be reverted back)

Reply:
here you are setting them but not using them till they are set again in the lower lines of code. I can put it back but it's not really testing anything.